editor/components/inserter-with-shortcuts/index.js

```diff
@@ -23,9 +23,12 @@ function InserterWithShortcuts( { items, isLocked, onInsert } ) {
 		return null;
 	}
 
-	const itemsWithoutDefaultBlock = filter( items, ( item ) =>
-		item.name !== getDefaultBlockName() || ! isEmpty( item.initialAttributes )
-	).slice( 0, 3 );
+	const itemsWithoutDefaultBlock = filter( items, ( item ) => {
```

Comment:
Is this function name still accurate? It's now doing more than just checking whether the block is the default block. To be fair, it wasn't even very accurate previously, because it was referring to the "unchanged default block", which is logic duplicated with [`wp.blocks.isUnmodifiedDefaultBlock`](https://github.com/WordPress/gutenberg/blob/a292eb514c7eb21c27323d7b1314df0cb5ca940f/blocks/api/utils.js#L27-L52).